demo_nodes_py/README.md

```diff
@@ -0,0 +1,280 @@
+## **What Is This?**
+
+This demo ...
+
+This package consists of ROS 2 nodes:
```

Comment:
Can remove "This demo ...",  and change the next line to,
"This package contains the following ROS 2 nodes:"

Reply:
Thank you for the review. Will follow up with the nits resolved shortly.

Reply:
Done.